web/client/components/widgets/builder/wizard/map/enhancers/__tests__/handleMapSelect-test.jsx

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2018, GeoSolutions Sas.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+const React = require('react');
+const ReactDOM = require('react-dom');
+const {createSink} = require('recompose');
+const expect = require('expect');
+import handleMapSelect from "../handleMapSelect";
+
+describe('handleMapSelect enhancer', () => {
+    beforeEach((done) => {
+        document.body.innerHTML = '<div id="container"></div>';
+        setTimeout(done);
+    });
+    afterEach((done) => {
+        ReactDOM.unmountComponentAtNode(document.getElementById("container"));
+        document.body.innerHTML = '';
+        setTimeout(done);
+    });
+    it('handleMapSelect enhancer callbacks', (done) => {
+        const sink = createSink( props => {
```

Comment:
This test doesn't check the issue is solved. If you remove your changes the test passes anyway. Please use my suggestion to run locally only this test ( `npm run continuoustest` allows you to run test on file change) and rewrite it to verify the real issue.